soh/assets/xml/GC_NMQ_PAL_F/overlays/ovl_En_Ganon_Mant.xml

```diff
@@ -1,21 +1,21 @@
 <Root>
-    <File Name="ovl_En_Ganon_Mant" BaseAddress="0x80A23D60" RangeStart="0x1218" RangeEnd="0x4118">
-        <Texture Name="gMantTex" OutName="mant" Format="rgba16" Width="32" Height="64" Offset="0x1218"/>
+    <File Name="ovl_En_Ganon_Mant" BaseAddress="0x80A0BCC0" RangeStart="0x11F8" RangeEnd="0x40F8">
```

Comment:
I'm confident about all the changes I made except for the `BaseAddress=` in these three files. I'm not sure the proper way to find them out.

Instead what I did was run ZAPD, and during the DL parsing it would throw an error saying the referenced texture address is below the base address.
In this case, the referenced texture is the first in each of these files, so I took the textures address, subtracted the offset, and used that as the `BaseAddress`. Obviously I tested the changes and everything works. Just want to make sure that anyone who understands it better can verify.

/cc @NEstelami